backend/fs.c

```diff
@@ -207,10 +207,13 @@ check_access(struct stat *st, uid_t uid, int amode)
 	if (st->st_uid == uid) {
 		if (amode == L9P_OREAD && st->st_mode & S_IRUSR)
 			return (true);
-		
+
 		if (amode == L9P_OWRITE && st->st_mode & S_IWUSR)
 			return (true);
-		
+
+		if (amode == L9P_ORDWR && st->st_mode & S_IRUSR && st->st_mode & S_IWUSR)
```

Comment:
Don't we still need these?

(Also, the getpwent and getgrouplist() calls need mods for thread-safety.  For now marking them not-thread-safe is probably the thing to do)